learn/experimental/vector_search.mdx

```diff
@@ -19,6 +19,10 @@ If using Meilisearch Cloud, navigate to your project overview and find "Experime
 
 ![A section of the project overview interface titled "Experimental features". There are two options: "Score details" and "Vector store". "Vector store" is turned on.](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vector-search/01-cloud-vector-store.png)
 
+<Capsule intent="note" title="Waiting list">
+To ensure proper onboarding and scaling of Meilisearch Cloud's latest AI-powered search offering, you must enter the waitlist before activating vector search.
+</Capsule>
+
 Alternatively, use [the `/experimental` route](/reference/api/experimental_features) to activate vector search during runtime:
```

Comment:
This applies up to v1.6 included

Reply:
Does that mean the experimental features route will be unavailable for v1.7?